src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -107,7 +107,11 @@ public static FormulaValue FromJson(JsonElement element, FormulaValueJsonSeriali
                     {
                         DateTime dt3 = element.GetDateTime();
                         return DateTimeValue.New(TimeZoneInfo.ConvertTimeToUtc(dt3));
-                    }                              
+                    }
+                    else if (formulaType is BlobType)
+                    {
+                        return FormulaValue.NewBlob(element.GetBytesFromBase64());
+                    }

```

Comment:
Is this consistent with Power Apps?

Double check against spec at: 

[https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-json](https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-json)

Reply:
I'll add a few tests

Reply:
Power Apps' JSON function _does not support_ serializing blobs. It does support serializing media (DType.Media, DType.Image), and it does as a data URI

Reply:
Can you  tag me on the new tests for this case?
I'm looking for something that calls JSON() function and verifies this new code doesn't create a divergent behavior with Power Apps.

Reply:
This part of the code is reached with Office365Outlook.GetAttachmentV2